maas/plugins/horizon_check.py

```diff
@@ -45,8 +45,8 @@ def check(args):
     OS_USERNAME = auth_details['OS_USERNAME']
     OS_PASSWORD = auth_details['OS_PASSWORD']
     OS_USER_DOMAIN_NAME = auth_details['OS_USER_DOMAIN_NAME']
-    HORIZON_URL = 'https://{ip}'.format(ip=args.ip)
-    HORIZON_PORT = '443'
+    HORIZON_URL = 'http://{ip}'.format(ip=args.ip)
+    HORIZON_PORT = '80'
```

Comment:
Do any changes need to go into the f5 config tool to facilitate this upstream change?  Perhaps that will come later which I think is acceptable, but figured I'd point it out just incase.